launcher.js

```diff
@@ -5,6 +5,7 @@ const LANG = {
     "en": {
         "Sponsored": "Sponsored",
         "Sponsored · Paid for by": "Sponsored · Paid for by",
+        "Suggestions": "Suggestions",
```

Comment:
The key name seems french to me, change to "Suggested" (?)

Reply:
Yes. It sounds French. But translation of Suggestions to English is "Suggestions". :D